scripts/data/germanquad/corpus.jsonl

```diff
@@ -0,0 +1,474 @@
+{"_id": "c40369", "text": "Aufzugsanlage\n\n=== Seilloser Aufzug ===\nAn der RWTH Aachen im Institut f\u00fcr Elektrische Maschinen wurde ein seilloser Aufzug entwickelt und ein Prototyp aufgebaut. Die Kabine wird hierbei durch zwei elektromagnetische Synchron-Linearmotoren angetrieben und somit nur durch ein vertikal bewegliches Magnetfeld gehalten bzw. bewegt. Diese Arbeit soll der Entwicklung von Aufzugsanlagen f\u00fcr sehr hohe Geb\u00e4ude dienen. Ein Ziel ist der Einsatz mehrerer Kabinen pro Schacht, die sich unabh\u00e4ngig voneinander steuern lassen. Bei Auswahl des Fahrtziels vor Fahrtantritt (d.\u00a0h. noch au\u00dferhalb des Aufzug) wird ein bestimmter Fahrkorb in einem der Aufzugssch\u00e4chte f\u00fcr die Fahrt ausgew\u00e4hlt, mit der sich der geplante Transport am schnellsten durchf\u00fchren l\u00e4sst. Der Platzbedarf f\u00fcr die gesamte Aufzugsanlage k\u00f6nnte somit um ein oder mehrere Sch\u00e4chte reduziert werden. Da die Kabinen seillos betrieben werden, ist ein Schachtwechsel ebenfalls denkbar. Hiermit k\u00f6nnen weitere Betriebsstrategien f\u00fcr die seillose Aufzugsanlage entwickelt werden, zum Beispiel ein moderner Paternosteraufzug mit unabh\u00e4ngig voneinander beweglichen Kabinen.\nIm Rahmen der Forschungen an dem seillosen Aufzug wird ebenfalls an der Entwicklung elektromagnetischer Linearf\u00fchrungen gearbeitet, um den Verschlei\u00df der seillosen Aufzugsanlage bei hohem Fahrkomfort zu minimieren. Weltweit wird an verschiedenen Forschungseinrichtungen an seillosen Antriebsl\u00f6sungen f\u00fcr Aufz\u00fcge gearbeitet. Otis betreibt zu diesem Zweck seit 2007 den ''Shibayama Test Tower''. ThyssenKrupp Elevator weihte 2017 im s\u00fcddeutschen Rottweil einen Testturm ein, in welchem die Technik des seillosen Aufzugs mit Synchron-Linearmotoren im Originalma\u00dfstab getestet wird. Der erste Aufzug dieses Typs soll 2020 in Berlin in Betrieb gehen."}
```

Comment:
This is the final processed corpus, correct? I think we should remove it from the repo & put it on huggingface
I can invite you to the MTEB org if you want to put it there

Reply:
sure, let's host the dataset there :)